packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -568,6 +568,36 @@ describe('I18n', () => {
         });
         expect(i18n.unformatCurrency('1,233.45', 'EUR')).toBe('1233.45');
       });
+
+      describe('fr locale', () => {
```

Comment:
Any other edge cases we'd like to have covered?

Reply:
Do we have to be more specific on the locale? `fr-FR` and `fr-CA` will probably have different behaviours?